docs/index.md

```diff
@@ -2,6 +2,24 @@
 
 _TAXonomic Profile Aggregation and STAndardisation_
 
+## About
+
+Taxpasta aims to produced `standardise`d taxon tables from taxonomic profiles, facilitating easier comparison of output of different tools. It can `merge` standardised profiles to generate per-tool multi-sample taxon tables, as well as generate `consensus` profiles across different tools.
```

Comment:
```suggestion
Taxpasta aims to produced `standardised` taxon tables from taxonomic profiles, facilitating easier comparison of output of different tools. It can `merge` standardised profiles to generate per-tool multi-sample taxon tables, as well as generate `consensus` profiles across different tools.
```